Firestore/core/src/firebase/firestore/model/document_key.h

```diff
@@ -21,9 +21,20 @@
 #include <memory>
 #include <string>
 
+#if defined(__OBJC__)
+#include "Firestore/Source/Model/FSTDocumentKey.h"
+#endif  // defined(__OBJC__)
+
 #include "Firestore/core/src/firebase/firestore/model/resource_path.h"
 #include "absl/strings/string_view.h"
 
+#if defined(__OBJC__)
+// Somehow, when ObjC header includes a C++ that includes that ObjC header, the
+// ObjC class declaration is not visible. So manually forward-declares the class
+// here.
```

Comment:
As above. (I suspect you just need to update the comment.)

Reply:
done